### PR TITLE
Fix of "Prevent nested ternaries in the code #1526"

### PR DESCRIPTION
**Pull request by nilscb**

### CODE REVIEW

File: react/.eslintrc [hunk 1]
```diff
@@ -37,6 +37,7 @@
   ],
   "rules": {
     "react-hooks/rules-of-hooks": "error", // For checking rules of hooks
-    "react-hooks/exhaustive-deps": "error" // For checking hook dependencies 
+    "react-hooks/exhaustive-deps": "error", // For checking hook dependencies
+    "no-nested-ternary": 2
```
Comment (hkfb) [inline react/.eslintrc:41]: What does "2" mean?

Comment (hkfb) [reply react/.eslintrc]: Can you use "error" like above? It's clearer.

**Review by hkfb — CHANGES_REQUESTED** [review]

File: react/src/lib/components/SubsurfaceViewer/components/Map.tsx [hunk 2]
```diff
@@ -1281,11 +1281,16 @@ function createViews(
                 const currentViewport: ViewportType =
                     views.viewports[deckgl_views.length];
 
-                const ViewType = currentViewport.show3D
-                    ? OrbitView
-                    : currentViewport.id === "intersection_view"
-                    ? IntersectionView
-                    : OrthographicView;
+                let ViewType:
```
[the remark this answers is not in the document]
Comment (nilscb) [reply react/src/lib/components/SubsurfaceViewer/components/Map.tsx:1284]: It already has a default as it set to OrbitView initially.

**Review by hkfb — APPROVED** [review]

**Review by hkfb — APPROVED** [review]

**Pull request merged**